public/sass/elements/forms/_form-multiple-choice.scss

```diff
@@ -8,19 +8,15 @@
   clear: left;
   position: relative;
 
-  padding: (8px $gutter-one-third 9px 50px);
+  padding: 0 0 0 38px;
   margin-bottom: $gutter-one-third;
 
-  cursor: pointer; // Encourage clicking on the label
-
    // remove 300ms pause on mobile
   -ms-touch-action: manipulation;
   touch-action: manipulation;
```

Comment:
The `touch-action` lines: do we want to move them into the `label` section as well? Ideally we’d minimise the code these are applied to, as (for example) they prevent double-tap zoom. (thanks to @nickcolley for pointing that out)